w10/d02_bb_models_collections/lesson_plan.md

```diff
@@ -276,7 +280,7 @@ touch app/assets/javascripts/backbone/collections/remindersCollection.js
 ```
 
 #### Models
-Remember all that complex ajax stuff we used to get information from a server? Well BB obfuscates all of that for you and makes it much easier. All we have to do is change up our model definition slightly! In `app/assets/javascripts/backbone/models/reminderModel.js`:
+Remember all that complex ajax stuff we used to get information from a server? Well BB abstracts all of that for you and makes it much easier. All we have to do is change up our model definition slightly! In `app/assets/javascripts/backbone/models/reminderModel.js`:
```

Comment:
obfuscate = hide with the intention to confuse.
abstract = hide the details so we don't have to worry about them